src/summary.rs

```diff
@@ -32,6 +36,10 @@ impl Summary {
         self.has_formatting_errors
     }
 
+    pub fn has_diff(&self) -> bool {
+        self.has_diff
+    }
+
```

Comment:
yeah, good point, wanted to stay consistent, but I guess it's really unnecessary